server.js

```diff
@@ -2,7 +2,7 @@ const express = require("express");
 const path = require("path");
```

Comment:
Here you can set up the babel/preset-env so you can use the standard JS import instead of the require import (makes the files more cohesive)